File: Easy/1319 - unique-number-of-occurrences/Solution.cpp
class Solution {
public:
    bool uniqueOccurrences(vector<int>& arr) {
        unordered_map<int,int>frq;
        for(auto i:arr)frq[i]++;
         unordered_map<int,int>frq2;
         for(auto [k,v]:frq){
            frq2[v]++;
            if(frq2[v]>1)
                return false;
            }
        return true;
    }
};

File: easy/1319 - unique-number-of-occurrences/Solution.py
class Solution(object):
    def uniqueOccurrences(self, arr):
        """
        :type arr: List[int]
        :rtype: bool
        """
        b=list(map(abs,arr))
       
        a=list(set(b))
        print(a)
        return not a==b
        